PFR/MoreRuzsaDist.lean

```diff
@@ -460,27 +460,31 @@ lemma ent_of_sub_smul_le {Y : Ω → G} [IsProbabilityMeasure μ] [Fintype G]
     (hX: Measurable X) (hY: Measurable Y) (hindep: IndepFun X Y μ) {a : ℤ} :
     H[X - a • Y; μ] - H[X; μ] ≤ 4 * |a| * d[X ; μ # Y ; μ] := by
 
-  obtain ⟨Ω', mΩ', μ', nX'₁, Y', X'₂, hμ', hindep', hnX'₁, hY', hX'₂, idX₁, idY, idX₂⟩
-    := independent_copies3_nondep hX.neg hY hX  μ μ μ
-  let X'₁ := - nX'₁
+  obtain ⟨Ω', mΩ', μ', X'₁, Y', X'₂, hμ', hindep', hX'₁, hY', hX'₂, idX₁, idY, idX₂⟩
+    := independent_copies3_nondep hX hY hX  μ μ μ
+  -- let X'₁ := - nX'₁
 
-  have inX₁Y : IndepFun nX'₁ Y' μ' := hindep'.indepFun (show 0 ≠ 1 by simp)
+  have iX₁Y : IndepFun X'₁ Y' μ' := hindep'.indepFun (show 0 ≠ 1 by simp)
   have iYX₂ : IndepFun Y' X'₂ μ' := hindep'.indepFun (show 1 ≠ 2 by simp)
-  have inX₁X₂ : IndepFun nX'₁ X'₂ μ' := hindep'.indepFun (show 0 ≠ 2 by simp)
-  have iX₁Y : IndepFun X'₁ Y' μ' := inX₁Y.comp measurable_neg measurable_id
+  have iX₁X₂ : IndepFun X'₁ X'₂ μ' := hindep'.indepFun (show 0 ≠ 2 by simp)
+  -- have iX₁Y : IndepFun X'₁ Y' μ' := inX₁Y.comp measurable_neg measurable_id
   have iX₂nY : IndepFun X'₂ (-Y') μ' := iYX₂.symm.comp measurable_id measurable_neg
 
-  have idX₁X₂' : IdentDistrib X'₁ X'₂ μ' μ' := by
-    refine IdentDistrib.trans ?_ idX₂.symm
-    rw [← neg_neg X]
-    simp_rw [X'₁]
-    exact IdentDistrib.neg idX₁
+  let φ : Fin 3 → G → G
+    | 0 => (fun x ↦ - x)
+    | 1 | 2 => id
```

Comment:
```suggestion
  let φ : Fin 3 → G → G := ![-id, id, id]
```
if you fancy

Reply:
We are working on it right now. Integrated it. Thanks a lot.